### PR TITLE
[CU-1zhbq08] Develop STE Hero Section 

### PR DESCRIPTION
**Pull request by Samanthatb1**
## Type of change
- [x] New feature 

## Note
- Issues with the logo SVG whitespace causes the page to overflow, encouraging other members to take a look

# How Has This Been Tested?
- [x] Checked responsiveness in developer tools, issues remain as previously mentioned

Screen Shot 2022-03-01 at 7 26 44 PM

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation/READ-ME/UMLS
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by jennchenn** [comment]
Task linked: [CU-1zhbq08 Develop STE hero section](https://app.clickup.com/t/1zhbq08)

File: waterloop-site/src/components/SustainableTech/Hero/Hero.tsx [hunk 1]
```diff
@@ -1,5 +1,93 @@
 import React from 'react';
+import styled from 'styled-components';
 
-const Hero: React.FC = () => <div>Hero</div>;
+import LandingImage from '../../../static/img/sustainable-tech/HeroLandingImage.svg';
+import GreenTechLogo from '../../../static/img/sustainable-tech/GreenTechLogo.svg';
 
+const LandingContainer = styled.div`
+  padding: 60px 0 30px;
```
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Hero/Hero.tsx:8]: Is this required? When I run the branch I see an empty blue space at the bottom, and removing this padding fixes that.

File: waterloop-site/src/components/SustainableTech/Hero/Hero.tsx [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: fit-content;
+`;
+
+/*  transform: scale(8); makes the logo
+bigger but doesnt solve the problem */
```
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Hero/Hero.tsx:41]: This should be deleted before merging.

File: waterloop-site/src/components/SustainableTech/Hero/Hero.tsx [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+
+const LandingContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow: hidden;
+`;
+
+const LandingImageContainer = styled.img`
+  height: 100vh;
```
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Hero/Hero.tsx:23]: More than height, I think `width` is more important. Currently, the SVG doesn't fill up the entire width of the screen. I would replace this with `width: 100%`

File: waterloop-site/src/components/SustainableTech/Hero/Hero.tsx [hunk 4]
```diff
@@ -1,5 +1,93 @@
...
+  overflow: hidden;
+  background-color: #e5f6fa;
```
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Hero/Hero.tsx:25]: Don't believe you need this?

File: waterloop-site/src/components/SustainableTech/Hero/Hero.tsx [hunk 5]
```diff
@@ -1,5 +1,93 @@
...
+  }
+`;
+
+const Hero: React.FC = () => (
+  <LandingContainer>
+    <LandingContent>
+      <MainContent id="home-scroll">
+        <GreenTech alt={'Green Tech Logo'} src={GreenTechLogo}></GreenTech>
+        <ButtonContainer as="a" href="http://Google_Form_Here">
+          Sign Up
```
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Hero/Hero.tsx:83]: This should be capitalized, to match the Figma design.

**Review by williamhpark — CHANGES_REQUESTED** [review]
I'm still trying to figure out the logo issue, but these are some other suggestions I have in the meantime. Edit: Was able to fix the SVG issue, refer to the comments.

File: waterloop-site/src/components/SustainableTech/Hero/Hero.tsx [hunk 6]
```diff
@@ -1,5 +1,81 @@
...
+    font-size: 17px;
+    padding: 10px 15%;
+  }
+`;
+
+const Hero: React.FC = () => (
+  <LandingContainer id="outer">
+    <MainContent id="home-scroll">
+      <GreenTech alt="Green Tech Logo" src={GreenTechLogo} />
+      <ButtonContainer as="a" href="https://tinyurl.com/uwgreentech">
```
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Hero/Hero.tsx:75]: Can you make clicking this open a new tab by adding `target="_blank" rel="noopener noreferrer"`?

**Review by williamhpark — APPROVED** [review]
Just one small change suggestion, then looks good to me!

**Review by eevanwong — COMMENTED** [review]
image

On my screen its a bit cut off :/. I have a relatively large monitor so I think in order to match the size of my monitor. The `100vh` height was not enough to maintain the aspect ratio of the original image. 

I've attempted other options to control the height, but it would be ideal that height was only `100vh`. 

In the `LandingContainer` div, I've tried to experiment with how it covers it (e.g `background-size: contain;` which leads to: 
image.

Other than that, everything else looks good! As a potential workaround, we could change the actual figma such that the goose is a bit farther up (but make it smaller) so that it's not evidently cropped.

**Review by eevanwong — CHANGES_REQUESTED** [review]
To avoid the cropping issue, a potential solution may be to adjust the figma component by moving the large goose at the front a little bit up and make it smaller so that there is more of the blue at the bottom.

This would allow us to be more forgiving in terms of cropping in different screens.

**Review by williamhpark — APPROVED** [review]

**Pull request merged**